pyproject.toml

```diff
@@ -144,11 +144,11 @@ semgrep = ">=1.34.0"
 types-certifi = "^2021.10.8"
 types-freezegun = "^1.1.7"
 types-mock = "^5.0.0.5"
-types-pkg-resources = "^0.1.3"
+types-setuptools = "^74.1.0.20240907"
```

Comment:
`types-pkg-resources` is yanked and must be replaced with `types-setuptools`[^1][^2]. The fact it was yanked also caused Poetry to crash thus making it impossible to run poetry lock.

[^1]: https://github.com/python/typeshed/issues/10340
[^2]: https://pypi.org/project/types-pkg-resources/0.1.2/